fastlane/lib/fastlane/fast_file.rb

```diff
@@ -24,6 +24,13 @@ def initialize(path = nil)
                 'you should turn off smart quotes in your editor of choice.'
       end
 
+      content.scan(/.*require (.*)/).each do |current|
```

Comment:
Are we looking for `.*require` because we want to check both `fastlane_require` and regular `require`?